apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/PipelineManager.java

```diff
@@ -277,7 +302,7 @@ private void handleJobFailure(PipelineJobRecordId jobRecordId, Exception excepti
         jobRecordStore.recordJobFailure(jobRecordId, new PipelineJobFailure(exception));
         jobsEnqueuedHandles.remove(jobRecordId);
       }
-      LOGGER.error("Handle job failure in Pipeline: " + exception.getMessage(), exception);
+      LOGGER.error("Job failure in Pipeline: " + exception.getMessage(), exception);
```

Comment:
Curious if you tried to log EMERGENCY here; if things go south in the ETL, not even sure why we try to wait/cleanup threads....the thing is hosed.

Reply:
Yeah I'm also not sure why them emphasis on the thread cleanup.. I assume it's to try and not leave us half-loaded and in a bad state, but we'd probably either rollback or reload in idempotent mode anyhow. Just trying to keep the general idea of the shutdown process in case it was important (since I have no idea if there was a good reason not to just kill everything the moment the pipeline chokes.)